forge-gui/res/cardsfolder/upcoming/imodane_the_pyrohammer.txt

```diff
@@ -0,0 +1,8 @@
+Name:Imodane, the Pyrohammer
+ManaCost:2 R R
+Types:Legendary Creature Human Knight
+PT:4/4
+T:Mode$ DamageDone | ValidSource$ Instant.YouCtrl,Sorcery.YouCtrl | ValidTarget$ Creature | ValidCause$ Spell.singleTarget | TargetRelativeToCause$ Creature.targetedBy | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ Whenever an instant or sorcery spell you control that targets only a single creature deals damage to that creature, NICKNAME deals that much damage to each opponent.
```

Comment:
`ValidSource$ Spell.numTargets EQ1`

Reply:
> `ValidSource$ Spell.numTargets EQ1`

ValidSource doesn't have `numTargets` 
thats why there is `ValidCause$ Spell.singleTarget`

Reply:
yea I meant cause
the properties mean different things